articles/architecture-scenarios/under-construction/index.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I believe this doc is not needed. I understand that you created it so the breadcrumb "Under Construction" leads here, but I don't think you even need this directory.

I'd take all these files https://github.com/auth0/docs/tree/breadcrumbs-fixes/articles/architecture-scenarios/under-construction
and move them under https://github.com/auth0/docs/tree/breadcrumbs-fixes/articles/architecture-scenarios